README.md

```diff
@@ -86,46 +87,95 @@ Ensure you have the following installed:
 
 Follow these steps to set up and run the project using Docker.
 
-1. **Build and Start the Container**
+1. **Build and Start the Containers**
 
     ```bash
-    # build the container
-    make docker-build
-    # start the container
-    make docker-start
+    # pull the supporting containers
+    make compose-pull
+
+    # build the main application image
+    make compose-build
+
+    # start the containers
+    make compose-up
     ```
 
 2. **Migrations and Superuser Creation**
 
     If this is your first time setting up the project, you’ll need to run migrations to set up the database schema and create an administrative user.
-    Note: `dj` is an alias for `django-admin`
 
     ```bash
     # ssh into the web container
     make docker-shell
 
     # Run database migrations
-    dj migrate
+    make migrate
 
     # Create a superuser for accessing the admin interface
-    dj createsuperuser
+    make createsuperuser
     ```
 
 3. **Start Django Inside the Container**
 
     Once the containers are running, you need to manually start Django from within the web container. This allows for running both the Django server and any additional background services (e.g., schedulers).
 
+    ⚠️ WARNING  
```

Comment:
You can still use the panel.
```suggestion
    > ⚠️ WARNING  
```

Reply:
Done, `make format` enforces that we put a chevron before the text on line 123 as well